types/react-router-redux/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "../tslint.json" }
+{
+    "extends": "../tslint.json",
+    "rules": {
+        "forbidden-types": false
```

Comment:
I need that because the [type definition for` redux`](https://github.com/reactjs/redux/blob/fd39b9d07f543725ce506ade2fb319d5020bad83/index.d.ts#L377) (which is packaged with `redux` itself) does use the type `Function`. I tried but couldn't remove the dependency on `redux` for `react-router-redux` type definition.